Practice/kruglov/home_task_4/PyLec4T6.py

```diff
@@ -0,0 +1,31 @@
+# 6 PyLecT6.py
+
+def del_find(el, mtx):
+    i = 0
+    x = []
+    for row in mtx: 
+        while i < len(row):
+            if el == row[i] and i not in x:
+                x.append(i)
+            i += 1
+        i = 0
+    x.sort()
+    for row1 in mtx:
+        cv = len(row)
+        while i < cv:
```

Comment:
You can just go through "x" here and remove row1[each] for each in x. Also it's better to start removal from the last column elements.

Reply:
Great idea! Start with the last elements, thanks!)